README.md

```diff
@@ -47,11 +47,16 @@ Station Core is configured using environment variables (see
 
 The following configuration options are shared by all Station commands:
 
-- `$ROOT_DIR`_(string; optional)_: Station stores logs and module state in this
+- `$CACHE_ROOT`_(string; optional)_: Station stores logs and module state in
+  this directory. Defaults to
```

Comment:
I think you have accidentally swapped the descriptions.

```suggestion
- `$CACHE_ROOT`_(string; optional)_: Station stores module caches in this
  directory. Defaults to
```

I would phrase this slightly differently:

_Station stores temporary files (e.g. cached data) in this directory. Defaults to_